webcompat/templates/layout.html

```diff
@@ -35,8 +37,9 @@
 <script src="{{ url_for('static', filename='js/lib/navbar.js') }}"></script>
 <script src="{{ url_for('static', filename='js/lib/autogrow-textfield.js') }}"></script>
 <script src="{{ url_for('static', filename='js/lib/bugform-validation.js') }}"></script>
+{%- if ab_active('exp') == 'form-v1' %}
```

Comment:
This was omitted here on purpose. I would rather override only things for `form-v2` version

Reply:
Also by reverting this the current website tests are not broken.

Reply:
Also keep in mind that not all the population would end up in an experiment branch (eg. private browsing etc)

Reply:
Also people that work on the project are exempt from the experiments.

Reply:
This might work better: https://github.com/johngian/webcompat.com/commit/975be048eff3d8c00d293d5628d2787b1a32bd1a

Also keeps the tests happy

Reply:
@johngian thank you for your suggestions and for the [proposed commit](https://github.com/johngian/webcompat.com/commit/975be048eff3d8c00d293d5628d2787b1a32bd1a). I committed your update and all tests passed successfully.